src/Cleave.vue

```diff
@@ -63,6 +63,10 @@ export default {
         this.cleave.destroy()
         this.cleave = new Cleave(this.$el, val)
       }
+    },
+    // watch the value to make sure external changes (not by input) are taken into account as well
+    value: function (newValue) {
+      this.cleave.setRawValue(newValue);
```

Comment:
Thank you, a little thing should be changed,  use `value(newValue)`

Reply:
@fundon Thanks for looking into this!

I believe I accidentally removed my original source repository, so I was not able to update my commit. I therefore created a new Pull Request instead: https://github.com/vue-bulma/cleave/pull/22. Please have another look there. I will close this Pull Request